service/spawn.go

```diff
@@ -137,7 +137,7 @@ func (uis *UIServer) spawnPage(w http.ResponseWriter, r *http.Request) {
 		SetupScriptPath              string
 		NewUILink                    string
 		ViewData
-	}{spawnDistro, spawnTask, maxHosts, settings.Spawnhost.UnexpirableHostsPerUser, settings.Spawnhost.UnexpirableVolumesPerUser, settings.Providers.AWS.MaxVolumeSizePerUser,
+	}{*spawnDistro, spawnTask, maxHosts, settings.Spawnhost.UnexpirableHostsPerUser, settings.Spawnhost.UnexpirableVolumesPerUser, settings.Providers.AWS.MaxVolumeSizePerUser,
```

Comment:
need nil check before dereference (although it seems like this page shouldn't fail to load if distro doesn't exist)

Reply:
Changed it back to using a regular distro and using a dereferenced pointer if it is not nil.